src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -310,62 +311,71 @@ class CityColumnChart extends Component {
+          chart={props.chart}
+          cities={props.cities}
+        />
+
+        <div aria-hidden="true">
+          <header className={style.titleWrapper}>
+            <h1 className={style.title}>
+              {props.chart.name}
+            </h1>
+
+            <Tooltip
+              text={props.chart.description}
+              borderColor={colorMedium}
+            >
+              <Icon
+                icon="questionMark"
+                className={style.aboutChartIcon}
+                size={22}
+                color={colorMedium}
+                title=""
+              />
+            </Tooltip>
+          </header>
+
+          <div className={style.metaWrapper}>
+            <Tooltip
+              borderColor={colorMedium}
+              text={firstIndicator.contextual
+                ? STRINGS.CONTEXTUAL_DEFINITION
+                : STRINGS.PERFORMANCE_DEFINITION
+              }
+            >
+              <Icon
+                className={style.indicatorTypeMark}
+                color={colorDark}
+                icon={firstIndicator.contextual ? 'contextualIndicator' : 'performanceIndicator'}
+                size={14}
+                title=""
+              />
+            </Tooltip>
+
+            {firstIndicator.lastUpdated && `Last updated ${firstIndicator.lastUpdated}`}
+          </div>
+
+          {isMultiple && (
+            <Legend
+              // Legend is our own HTML so we can style and position it with CSS
+              className={style.legendWrapper}
+              series={getSeries(props)}
             />
-          </Tooltip>
-        </div>
-
-        <div className={style.metaWrapper}>
-          <Tooltip
-            borderColor={colorMedium}
-            text={firstIndicator.contextual
-              ? STRINGS.CONTEXTUAL_DEFINITION
-              : STRINGS.PERFORMANCE_DEFINITION
-            }
-          >
-            <Icon
-              className={style.indicatorTypeMark}
-              color={colorDark}
-              icon={firstIndicator.contextual ? 'contextualIndicator' : 'performanceIndicator'}
-              size={14}
-              title=""
-            />
-          </Tooltip>
+          )}
 
-          {firstIndicator.lastUpdated && `Last updated ${firstIndicator.lastUpdated}`}
-        </div>
-
-        {isMultiple && (
-          <Legend
-            // Legend is our own HTML so we can style and position it with CSS
-            className={style.legendWrapper}
-            series={getSeries(props)}
-          />
-        )}
-
-        <div className={style.axisTitle}>
-          {props.chart.axisTitle}
-        </div>
+          <div className={style.axisTitle}>
+            {props.chart.axisTitle}
+          </div>
 
-        <div className={style.chartWrapper}>
-          <div id={this.chartDivId} />
+          <div className={style.chartWrapper}>
+            <div id={this.chartDivId} />
+          </div>
         </div>
-      </div>
+      </article>
```

Comment:
This looks like a lot changes but it's just indenting and a few new semantic tags, and the new `<ChartTable>`